### ISSUE TITLE
Wrong copyright definition in license file

### ISSUE THREAD

**Issue by MiErnst** [bigskysoftware/htmx#1834]
The [license file of the htmx project](https://github.com/bigskysoftware/htmx/blob/master/LICENSE) contains a copyright mark with "Copyright (c) 2020, Big Sky Software" but this seems to be wrong. Besides that the date of the copyright is outdated (should be something like 2020-2023), the project does not have a Contributor License Agreements that grants Big Sky Software the right to publish contributions under their Copyright.
Google for example only accepts PRs if the their Contributor License Agreement is signed: https://cla.developers.google.com/about/google-corporate
As this is missing I think you have to list all the contributors of the project within the Licence file with their year of first contribution to current year.

I'm not a lawyer or expert in this field but as far as I understand it, no company can use htmx without listing all contributors in the licence file.

Alternative fix would be to rewrite htmx from scratch, create a new license file with a copyright mark, create a contribution process that includes a signed Contributor License Agreement before accepting PRs and publish everything under a new major version.

**Comment by MiErnst** [comment]
You are right it isn‘t lost but it shows some kind of quality to keep it up to date.

Nevertheless I have to disagree to the sentence with the OSS license. BSS does not automatically get the right, to publish it under their copyright only. That is exactly described here: https://opensource.stackexchange.com/a/5855

„When you accept a contribution, two things happen: 1. the contribution is under the existing project license 2. the copyright of this contribution is that of the contributor.

Therefore the copyright of the codebase as a whole is now shared between you and the contributor. “…


So from my point of perspective you need a CLA or another process to publish all contributions under the copyright note of BSS. Alternative is to add an authors file to the project and reference to the authors in the license file.

**Comment by MiErnst** [comment]
Just verified it with our company lawyer that it is OK for htmx developers to provide the license together with the given copyright notice (Big Sky Software only) as the license obligates htmx users to publish it with the given copyright notice defined in the license. Nevertheless the Copyright notice have to be fixed by the project I think so. So the issue is still valid but seems not be so harmful to htmx projects as my initial thought.

**Comment by Delapouite** [comment]
As the license has been changed recently, I think this issue can now be closed: https://github.com/bigskysoftware/htmx/commit/e16f1865a494b6281f8a44ed0db5338e718b3f07

**Comment by 1cg** [comment]
fixed
